libr/core/cmd_anal.c

```diff
@@ -39,9 +39,81 @@ static bool anal_is_bad_call(RCore *core, ut64 from, ut64 to, ut64 addr, ut8 *bu
 }
 #endif
 
+static void cmd_esil_mem(RCore *core, const char *input) {
```

Comment:
make it return bool. the function have several return points. try to return false if it fails